.ci/Jenkinsfile_coverage

```diff
@@ -5,89 +5,26 @@ kibanaLibrary.load() // load from the Jenkins instance
 
 kibanaPipeline(timeoutMinutes: 240) {
   catchErrors {
+    def timestamp = new Date(currentBuild.startTimeInMillis).format("yyyy-MM-dd'T'HH:mm:ss'Z'", TimeZone.getTimeZone("UTC"))
     withEnv([
-      'CODE_COVERAGE=1', // Needed for multiple ci scripts, such as remote.ts, test/scripts/*.sh, schema.js, etc.
+      "TIME_STAMP=${timestamp}",
```

Comment:
Why is it two words?

Reply:
two words?  Perhaps you mean the groovy to bash interpolation?

Reply:
The environment variable you're setting is `TIME_STAMP`, when `TIMESTAMP` seems more appropriate.

Reply:
I'll make this change chief